### PR TITLE
Merge ThrowableUtils into ThrowableUtil.

### PR DESCRIPTION
**Pull request by normanmaurer**
Motivation:

We should merge ThrowableUtils into ThrowableUtil as this name is more consistent with the naming of utility classes in netty.

Modifications:

Merge classes.

Result:

More consistent naming

### CODE REVIEW

**Comment by normanmaurer** [comment]
@Scottmitch PTAL

**Comment by Scottmitch** [comment]
lgtm

File: common/src/main/java/io/netty/util/internal/ThrowableUtil.java [hunk 1]
```diff
@@ -26,4 +30,26 @@ private ThrowableUtil() { }
...
+    /**
+     * Gets the stack trace from a Throwable as a String.
+     *
+     * @param cause the {@link Throwable} to be examined
+     * @return the stack trace as generated by {@link Throwable#printStackTrace(java.io.PrintWriter)} method.
+     */
+    public static String stackTraceToString(Throwable cause) {
+        ByteArrayOutputStream out = new ByteArrayOutputStream();
+        PrintStream pout = new PrintStream(out);
+        cause.printStackTrace(pout);
```
Comment (netkins) [inline common/src/main/java/io/netty/util/internal/ThrowableUtil.java]: CRITICAL Use a logger to log this exception. [rule](https://garage.netty.io/sonarqube/coding_rules#rule_key=squid%3AS1148)

**Comment by normanmaurer** [comment]
Cherry-picked into 4.0 (9d29c50) and 4.1 (9d29c50)

**Pull request closed without merging**